### DISCUSSION TITLE
‌‌‌Thank you for your exploration of cross-platform capabilities!

### DISCUSSION THREAD

**Discussion by Daydreamer-riri** [DevToys-app/DevToys#1233] [General]
‌‌I am someone who is very particular about the native application style. As a user of Windows, I pay a lot of attention to the Mica effect. Your [blog](https://devtoys.app/blog/the-journey-to-devtoys-2.0) has been very inspiring to me. Thank you for your exploration!

**Comment by veler** [comment]
You're welcome!
